packages/annotations/README.md

```diff
@@ -12,6 +12,10 @@ npm install @wordpress/annotations --save
 
 _This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats)._
 
+## Getting Started
+
+You need to include `wp-annotations` as a dependency of the JavaScript file in which you wish to use the Annotations API.
+
 ## Usage
 
-<br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
```

Comment:
Any reason for the "code is poetry" image removal? I think it's used in all the package readmes.

Reply:
I’ll open a new PR to re-add this wasn’t intentional!

Reply:
Thanks :)